verticapy/machine_learning/memmodel/cluster.py

```diff
@@ -320,6 +508,120 @@ class BisectingKMeans(Clustering, Tree):
         squares.
     p: int, optional
         The p corresponding to one of the p-distances.
+
+    Examples
+    --------
+
+    **Initalization**
+
+    Import the required module.
+
+    .. ipython:: python
+            :suppress:
+
+        from verticapy.machine_learning.memmodel.cluster import BisectingKMeans
+
+    A BisectingKMeans model is defined by its cluster centroids,  left and right child node id's of given node. In this example, we will use the following:
+
+    .. ipython:: python
+            :suppress:
+
+        clusters = [[0.5, 0.6], [1, 2], [100, 200], [10, 700], [-100, -200]]
+        children_left = [1, 3, None, None, None]
+        children_right = [2, 4, None, None, None]
+
+
+    Let's create a :py:mod:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans` model.
+
+    .. ipython:: python
+            :suppress:
+
+        model_bkm = BisectingKMeans(clusters, children_left, children_right)
+
+    Create a dataset.
+
+    .. ipython:: python
+            :suppress:
+
+        data = [[2, 3]]
+
+    **Making In-Memory Predictions**
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans.predict` method to do predictions
+
+    .. ipython:: python
+            :suppress:
+
+        model_bkm.predict(data)[0]
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans.predict_proba` method to compute the predicted probabilities for each cluster
+
+    .. ipython:: python
+            :suppress:
+
+        model_bkm.predict_proba(data)
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans.transform` method to compute the distance from each cluster
+
+    .. ipython:: python
+            :suppress:
+
+        model_bkm.transform(data)
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans.to_graphviz` method to generate code for a `Graphviz <https://graphviz.org/>`_ tree
+
+    .. ipython:: python
+            :suppress:
+
+        model_bkm.to_graphviz()
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.cluster.BisectingKMeans.plot_tree` method to draw the input tree.
+
+    .. code-block:: python
+
+        model_bkm.plot_tree()
+
+    .. ipython:: python
+        :suppress:
+
+        res = model_bkm.plot_tree()
+        res.render(filename='figures/machine_learning_cluster_bisecting_kmeans', format='png')
+
+    .. image:: SPHINX_DIRECTORY/figures/machine_learning_cluster_bisecting_kmeans.png
```

Comment:
the image directive uses source as the base path. so the image path should be realtive.

.. image:: ../figures/machine_learning_cluster_bisecting_kmeans.png